must-gather/Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM registry.svc.ci.openshift.org/openshift/release:golang-1.12 AS builder
-WORKDIR /go/src/github.com/openshift/ptp-operator
+FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.6 AS builder
+WORKDIR /go/src/github.com/openshift/ptp-operator/must-gather
 COPY . .
 
-FROM quay.io/openshift/origin-must-gather:4.3.0
+FROM registry.svc.ci.openshift.org/ocp/4.7:must-gather
```

Comment:
What is the reason for two different Dockerfiles in this directory?  If this one is meant to be for external building by hand for developers, and not used in CI or ART builds, then it can stay as it was (probably with a later version).

Reply:
@yselkowitz you got it right, the Dockerfile w/o suffix is used for external building.
The one w suffix Dockerfile.rhel7 is used for ART building.